src/main/java/Main.java

```diff
@@ -59,6 +69,52 @@ public static void main(String[] args) throws InterruptedException {
 //        viewManagerModel.setActiveView(customTierListView.viewName);
 //        viewManagerModel.setActiveView(tierListView.viewName);
         viewManagerModel.firePropertyChanged();
+      
+        FollowState testing = new FollowState("terryfufu", "lt_rui", false);
```

Comment:
May have to write tests for the integration between this use case and the others in a dedicated testing folder